src/tools/rbd/action/MirrorGroup.cc

```diff
@@ -0,0 +1,669 @@
+                  << "    last_update: " << utils::timestr(
+                    site.last_update) << std::endl;
+        std::cout << "    images:" << std::endl;
+        bool first_image = true;
+        for (auto &[p, image_status] : site.mirror_images) {
+          if (!first_image) {
+            std::cout << std::endl;
+          }
+          first_image = false;
+          // TODO: resolve pool_id/global_image_id into pool_name/image_name?
+          std::cout << "      image:       " << p.first << "/" << p.second << "\n"
+                    << "      state:       " << utils::mirror_image_site_status_state(
+                                                  image_status) << "\n"
+                    << "      description: " << image_status.description << "\n";
+        }
+      }
+    }
+    if (!snaps.empty()) {
+      std::cout << "  snapshots:" << std::endl;
+
+      bool first_snap = true;
+      for (auto &snap : snaps) {
+        if (!first_snap) {
+          std::cout << std::endl;
+        }
+        first_snap = false;
+        std::cout << "    " << snap.name;
+      }
+      std::cout << std::endl;
+    }
+  }
+
+  return 0;
+}
+
+void get_snapshot_arguments(po::options_description *positional,
+                            po::options_description *options) {
+  add_group_spec_options(positional, options);
+  at::add_snap_create_options(options);
+}
+
+int execute_snapshot(const po::variables_map &vm,
+                     const std::vector<std::string> &ceph_global_init_args) {
+  size_t arg_index = 0;
+  std::string pool_name;
+  std::string namespace_name;
+  std::string group_name;
+
+  int r = utils::get_pool_generic_snapshot_names(
+    vm, at::ARGUMENT_MODIFIER_NONE, &arg_index, at::POOL_NAME, &pool_name,
+    &namespace_name, GROUP_NAME, "group", &group_name, nullptr, true,
+    utils::SNAPSHOT_PRESENCE_NONE, utils::SPEC_VALIDATION_FULL);
+  if (r < 0) {
+    return r;
+  }
+
+  uint32_t flags;
+  r = utils::get_snap_create_flags(vm, &flags);
+  if (r < 0) {
+    return r;
+  }
+
+  librados::Rados rados;
+  librados::IoCtx io_ctx;
+
+  r = utils::init(pool_name, namespace_name, &rados, &io_ctx);
+  if (r < 0) {
+    return r;
+  }
+
+  r = validate_mirroring_enabled(io_ctx, group_name);
+  if (r < 0) {
+    return r;
+  }
+
+  librbd::RBD rbd;
+  std::string snap_id;
+  r = rbd.mirror_group_create_snapshot(io_ctx, group_name.c_str(), flags,
+                                       &snap_id);
+  if (r < 0) {
+    std::cerr << "rbd: error creating snapshot: " << cpp_strerror(r)
+              << std::endl;
+    return r;
+  }
+
+  std::cout << "Snapshot ID: " << snap_id << std::endl;
+  return 0;
+}
+
+Shell::Action action_enable(
+  {"mirror", "group", "enable"}, {},
+  "Enable RBD mirroring for an group.", "",
+  &get_arguments_enable, &execute_enable);
+Shell::Action action_disable(
+  {"mirror", "group", "disable"}, {},
+  "Disable RBD mirroring for an group.", "",
```

Comment:
```suggestion
  "Disable RBD mirroring for a group.", "",
```

Reply:
Adding a comment here to highlight this issue again. I it was @jagombar who posted a PR for this but I cannot find it.